YoutubeExplode/YoutubeClient.Channel.cs

```diff
@@ -30,6 +30,9 @@ public async Task<string> GetChannelIdAsync(string username)
         {
             username.GuardNotNull(nameof(username));
 
+            if (!ValidateUsername(username))
+                throw new ParseException("Could not parse channel ID.");
```

Comment:
Please change the type to `ArgumentException` and format it as `Invalid YouTube username [{username}].` so it's consistent with other instances of this exception.